.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,88 @@
+name: Pipeline [test -> build -> deploy]
+
+on:
+  push:
+    branches:
+      - main
+      # - '**'
+
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment
+        type: choice
+        required: true
+        options:
+          - dev
+          - prod
+          - staging
+        default: 'dev'
+      docker_registry:
+        description: Docker registry
+        required: true
+        default: 'ghcr.io'
+        type: choice
+        options:
+          - 'ghcr.io'
+          - 'quay.io'
+      registry_org:
+        description: Docker registry organisation
+        required: true
+        default: 'ministryofjustice'
+        type: choice
+        options:
+          - 'ministryofjustice'
```

Comment:
It could be confusing have 2 options for this - might be better to just stick to the `ghcr.io` and `ministryofjustice`.

Reply:
This, I think, is something you discussed with @Sandhya1874 - definitely worth reviewing.